app/services/analytics.rb

```diff
@@ -86,6 +86,7 @@ def browser
   IDV_PHONE_RECORD_VISIT = 'IdV: phone of record visited'.freeze
   IDV_REVIEW_COMPLETE = 'IdV: review complete'.freeze
   IDV_REVIEW_VISIT = 'IdV: review info visited'.freeze
+  IDV_VERIFICATION_ATTEMPT_CANCELLED = 'IdV: verification attempt cancelled'.freeze
```

Comment:
If this event can only be called by a user who is pending verification via USPS, perhaps we should add that to the event name, like `IdV: USPS verification attempt cancelled`? If it can be called by other types of users, then we probably want to capture the type of pending verification in the controller.

Reply:
From what I can tell, this event can only be triggered by someone clicking the link "clear my information and start over", so I think it makes sense to name the constant and event accordingly. Maybe something like `IDV_ADDRESS_VERIFICATION_RESTARTED = 'IdV: address verification restarted'`?